src/Reports/SubsiteReportWrapper.php

```diff
@@ -2,32 +2,31 @@
 
 namespace SilverStripe\Subsites\Reports;
 
+use SilverStripe\Forms\DropdownField;
 use SilverStripe\Forms\FieldList;
-use SilverStripe\Forms\TreeMultiselectField;
 use SilverStripe\Reports\ReportWrapper;
 use SilverStripe\Subsites\Model\Subsite;
+use SilverStripe\Subsites\State\SubsiteState;
 
 /**
  * Creates a subsite-aware version of another report.
  * Pass another report (or its classname) into the constructor.
  */
 class SubsiteReportWrapper extends ReportWrapper
 {
-
     /**
      * @return FieldList
      */
     public function parameterFields()
     {
         $subsites = Subsite::accessible_sites('CMS_ACCESS_CMSMain', true);
-        $options = $subsites->toDropdownMap('ID', 'Title');
+        $options = $subsites->map()->toArray();
```

Comment:
I think an "all" option should be added, which should be the default value. Given that we've removed the ability to select all subsites (because we can't set multiple subsite IDs) I think this is a reasonable compromise.

To add that ability, you can do this:
```php
public function beforeQuery($params)
{
    if (/*all subsites*/) {
        Subsite::disable_subsite_filter(true);
    }
}
```
and then obviously the reverse in `afterQuery()`.

The option could have, for example, a value of `-1` in the dropdown to make it clearly distinct.

Reply:
Updated